distributed/worker.py

```diff
@@ -1933,11 +1935,12 @@ def select_keys_for_gather(self, worker, dep):
 
                 if cause:
                     self.startstops[cause].append(
-                        (
-                            "transfer",
-                            start + self.scheduler_delay,
-                            stop + self.scheduler_delay,
-                        )
+                        {
+                            "action": "transfer",
+                            "start": start + self.scheduler_delay,
+                            "stop": stop + self.scheduler_delay,
+                            "worker": worker,
```

Comment:
The term worker here miight be a little bit ambiguous about which side of the transfer it refers to.  Maybe `sender` or `source`?  Not a huge deal either way, just an idle comment.

Reply:
Yeah that makes sense! My preference would be `source`.

Reply:
```suggestion
                            "source": worker,
```